src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -445,7 +441,8 @@ class CornerstoneViewport extends Component {
 
         // Display the first image
         cornerstone.displayImage(element, image, viewport);
-
+        const endLoadHandler = cornerstoneTools.loadHandlerManager.getEndLoadHandler();
+        endLoadHandler(element);
```

Comment:
We should guard this. `endLoadingHandler` might be undefined

Reply:
Done